src/components/common/TmSessionSignUpSeed.vue

```diff
@@ -115,12 +115,16 @@ export default {
           name: this.signup.signUpName,
           network: this.network
         })
-        this.$router.push({
-          name: "portfolio",
-          params: {
-            networkId: this.networkSlug
-          }
-        })
+        if (this.isExtension) {
```

Comment:
better solution would be make the path a property, but that would require a huge refactor so this solution is good.